main.go

```diff
@@ -121,6 +127,8 @@ func resolveConfig() (*config.Config, bool) {
 			conf.Verbose = verbose
 		case "role":
 			conf.Roles = roleFullnames
+		case "once":
+			conf.OutputOnce = *outputOnce
```

Comment:
OutputOnce is not an agent configuration but a CLI program behavior like printVersion (for `-version`), so we should not include OutputOnce to Config. I think it would be good to extend the return value of resolveConfig to pass `-once` flags, instead of appending it to conf.